### PR TITLE
adding missing routes

### PR DESCRIPTION
**Pull request by antoinemeyer**
@kvhswong please review.
added routes related to latest advisor-api changes

### CODE REVIEW

**Comment by antoinemeyer** [comment]
@bondricek @eweise @LowellBateman @kvhswong  @johntowell  please review

File: api/v2/advisor/client/billing/fees.raml [hunk 1]
```diff
@@ -33,6 +33,22 @@ get:
...
     description: |
+      ### Get the Fee Schedules
+      Return all the fee schedules for the fee schedule group
+    responses:
+      200:
+        body:
+          application/json:
+             example: !include examples/multipleFeeSchedules.res.json
+             schema: !include schema/mulitpleFeeSchedules.json
+/{groupKey}/active:
```
Comment (eweise) [inline api/v2/advisor/client/billing/fees.raml]: where does the groupKey come from?

Comment (antoinemeyer) [inline api/v2/advisor/client/billing/fees.raml]: it's "derived from the group name" :)

Comment (eweise) [inline api/v2/advisor/client/billing/fees.raml]: Is there an endpoint that provides it?

Comment (eweise) [inline api/v2/advisor/client/billing/fees.raml]: In the POST I see "group": {
      "description": "The group that this fee schedule belongs to"}
Does it derive from the description?

Comment (bondricek) [inline api/v2/advisor/client/billing/fees.raml]: it should come back in the get /v2/{advisorId}/client/billing/fees which will return all the fee schedules for the firm.

**Comment by eweise** [comment]
I'm good :+1:

**Pull request merged**